run

```diff
@@ -157,6 +157,8 @@ while len(args) > 0 and args[0].startswith("-"):
         if redirect_dir is True:
             cfg.add_mntopt("redirect_dir=off")
             redirect_dir = False
+    elif args[0] == "--skip-layer-test":
+        cfg.set_skip_layer_test(True)
```

Comment:
I don't mind this change, but I rather you don't add this option and set_skip_layer implicitly when using external mount helper.
However... I don't think you should skip layer check at all. 
Nowadays, layer check for --samefs or --verify checks that all files are on the same layer as the overlay mount.
I think that should be the case with fuse-overlayfs, so please double check if you need this patch at all

Reply:
yes you are right, I don't need this change if I use `--verify`, I am going to drop it